### PR TITLE
docs(zombies): Remove hydrate command, add zombie-specific commands

### PR DESCRIPTION
**Pull request by robzienert**

### CODE REVIEW

File: guides/runbooks/orca-zombie-executions.md [hunk 1]
```diff
@@ -36,80 +36,19 @@ If you've enabled the zombie check, set an alert on the metric `queue.zombies`,
 
 # Remediation
 
-## Rehydrate the Queue
+You can run this command to cancel a zombie execution via the Orca admin API. 
+This method is preferrable to the `forceCancelExecution` admin command, since it is more obvious for end-users why the execution was canceled.
```
Comment (dorbin) [inline guides/runbooks/orca-zombie-executions.md]: Since you've removed `forceCancelExecution` from this doc, is it still worth citing here?

**Review by dorbin — APPROVED** [review]

File: guides/runbooks/orca-zombie-executions.md (hunk 1 above)
Comment (robzienert) [reply guides/runbooks/orca-zombie-executions.md]: Probably not - will update.

Comment (317brian) [inline guides/runbooks/orca-zombie-executions.md]:
```suggestion

Comment (317brian) [reply guides/runbooks/orca-zombie-executions.md]: removing this line and then merging the PR

**Review by 317brian — APPROVED** [review]

File: guides/runbooks/orca-zombie-executions.md [hunk 2]
```diff
@@ -36,80 +36,18 @@ If you've enabled the zombie check, set an alert on the metric `queue.zombies`,
 
 # Remediation
 
-## Rehydrate the Queue
+You can run this command to cancel a zombie execution via the Orca admin API.
```
Comment (317brian) [inline guides/runbooks/orca-zombie-executions.md]:
```suggestion

**Pull request merged**